src/engine/enginevumeter.cpp

```diff
@@ -93,6 +99,24 @@ void EngineVuMeter::process(CSAMPLE* pIn, const int iBufferSize) {
     } else {
         --m_peakDuration;
     }
+
+    if (clipped & 1) {
```

Comment:
Since it will remove the redundant calulation using m_peakDuration, I still prefer the change.
But since it is not that important a comment will work as well.